core/src/main/scala/eu/shiftforward/apso/Implicits.scala

```diff
@@ -565,12 +565,25 @@ object Implicits {
   final implicit class ApsoCloseable[U <: AutoCloseable](val res: U) extends AnyVal {
 
     /**
-     * Uses this resouce and closes it afterwards.
+     * Uses this resource and closes it afterwards.
      * @param f the block of code to execute using this resource
      * @tparam T the return type of the code block.
      * @return the value returned by the code block.
      */
+    @deprecated("This will be removed in a future version. Please use `tryUse()` instead", "2017/10/31")
```

Comment:
This method still makes sense for me, since in many of the cases I do not explicitly want to handle exceptions. You can change its implementation to use `TryWith(res)(f).get` in order for it to use the new code.

Reply:
Altough I prefer having safer alternatives on this library, I have no problem in not deprecating this method... 
Removed notice and replace code in 17ffe8c